avocado/utils/pci.py

```diff
@@ -52,7 +52,7 @@ def get_pci_addresses():
     addresses = []
     cmd = "lspci -D"
     for line in process.system_output(cmd).splitlines():
-        if "PCI bridge" not in line and "System peripheral" not in line:
+        if '06' not in get_pci_prop(line.split()[0], 'Class'):
```

Comment:
Shouldn't this be `get_pci_prop().starts_with("06")`? Because there are devices like `0106` that are not bridges.

Reply:
You're reviewing a closed PR. See #2380